app/src/main/res/values/strings.xml

```diff
@@ -4,15 +4,15 @@
 
     <!-- Login activity -->
     <string name="app_title">beeline</string>
-    <string name="tag_line">Direct path to highest mutual attraction.</string>
+    <string name="tag_line">Your direct path to highest mutual attraction.</string>
```

Comment:
That's the branding tagline. It cannot be edited.

The tagline goes with all the marketing.

There is an argument that calling out the user with "your" is a nice touch. Though because it is the tagline, e.g. meant for a business card and a poster at a convention, it's already pretty wordy. Adding "your" makes it to my eyes so wordy that I'm less likely to actually read it when I'd path a convention booth.